### PR TITLE
Thymeleaf

### PR DESCRIPTION
**Pull request by gaurav-dogra**

### CODE REVIEW

File: src/main/java/gmailgdogra/DtoWrapper.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package gmailgdogra;
+
+import java.util.List;
+
+public class DtoWrapper {
```
Comment (alfabravo2013) [inline src/main/java/gmailgdogra/DtoWrapper.java:5]: The idea of a DTO wrapper is very new to me. Why do you need it?

File: src/main/resources/templates/shift-info.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org" lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Excel Parsing Web App</title>
+    <link th:href="@{main.css}" rel="stylesheet"/>
+</head>
+<body>
+<span th:text="${msg}"></span>
+<h2>Select Shift information for each officer</h2>
```
Comment (alfabravo2013) [inline src/main/resources/templates/shift-info.html:10]: it's a good idea to observe the hierarchy of headers. You don't have header 1. If you don't like its style, you can alays change it using css.

**Review by alfabravo2013 — APPROVED** [review]
Good job.

File: src/main/resources/templates/shift-info.html (hunk 2 above)
Comment (gaurav-dogra) [reply src/main/resources/templates/shift-info.html:10]: fixed

File: src/main/java/gmailgdogra/DtoWrapper.java (hunk 1 above)
Comment (gaurav-dogra) [reply src/main/java/gmailgdogra/DtoWrapper.java:5]: one of the easy way to return a list from Thymeleaf view. it be accessed in controller with 
@ModelAttribute DtoWrapper dtoWrapper

Comment (alfabravo2013) [reply src/main/java/gmailgdogra/DtoWrapper.java:5]: Ok, got it

**Pull request merged**